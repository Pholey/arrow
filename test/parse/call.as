method();

method()(10, 620);

method(560, 20);
