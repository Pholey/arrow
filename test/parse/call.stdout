{
  "tag": "Module",
  "span": "test/parse/call.as:1-5",
  "statements": [
    {
      "tag": "Call",
      "span": "test/parse/call.as:1:7-9",
      "operand": {
        "tag": "Identifier",
        "span": "test/parse/call.as:1:1-7",
        "text": "method"
      },
      "arguments": []
    },
    {
      "tag": "Call",
      "span": "test/parse/call.as:3:9-18",
      "operand": {
        "tag": "Call",
        "span": "test/parse/call.as:3:7-9",
        "operand": {
          "tag": "Identifier",
          "span": "test/parse/call.as:3:1-7",
          "text": "method"
        },
        "arguments": []
      },
      "arguments": [
        {
          "tag": "Argument",
          "span": "test/parse/call.as:3:10-12",
          "value": {
            "tag": "Integer",
            "span": "test/parse/call.as:3:10-12",
            "value": "10"
          }
        },
        {
          "tag": "Argument",
          "span": "test/parse/call.as:3:14-17",
          "value": {
            "tag": "Integer",
            "span": "test/parse/call.as:3:14-17",
            "value": "620"
          }
        }
      ]
    },
    {
      "tag": "Call",
      "span": "test/parse/call.as:5:7-16",
      "operand": {
        "tag": "Identifier",
        "span": "test/parse/call.as:5:1-7",
        "text": "method"
      },
      "arguments": [
        {
          "tag": "Argument",
          "span": "test/parse/call.as:5:8-11",
          "value": {
            "tag": "Integer",
            "span": "test/parse/call.as:5:8-11",
            "value": "560"
          }
        },
        {
          "tag": "Argument",
          "span": "test/parse/call.as:5:13-15",
          "value": {
            "tag": "Integer",
            "span": "test/parse/call.as:5:13-15",
            "value": "20"
          }
        }
      ]
    }
  ]
}
